core/src/main/java/software/amazon/awssdk/internal/CustomBackoffStrategy.java

```diff
@@ -1,20 +0,0 @@
-/*
- * Copyright 2010-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License").
- * You may not use this file except in compliance with the License.
- * A copy of the License is located at
- *
- *  http://aws.amazon.com/apache2.0
- *
- * or in the "license" file accompanying this file. This file is distributed
- * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
- * express or implied. See the License for the specific language governing
- * permissions and limitations under the License.
- */
-
-package software.amazon.awssdk.internal;
-
-public abstract class CustomBackoffStrategy {
```

Comment:
Has this been replaced by something else? Is this not what we provide customers to use to provide custom backoff?

Reply:
It wasn't in use, but I figured if we wanted to support it we'd handle it in the retry refactor. Thoughts?

Reply:
Guess I'm fine waiting til retry refactor.